english/play-with-dog/dog.md

```diff
@@ -1,12 +1,14 @@
 You walk down Valencia and see a little french bulldog appear around the corner.
 
-You follow her down 18th and end up at Dolores Park. You dodge around all of the 
+You follow her down 18th and end up at Dolores Park. You dodge around all of the
 
-young 20-somethings day drinking and finally catch up to the dog. 
+young 20-somethings day drinking and finally catch up to the dog.
 
-She lies on her belly. 
+She lies on her belly.
 
 You [take a selfie](../selfie/selfie.md).
 
-You have actually followed the dog embodiment of Medusa, 
-and you made eye contact. [You can no longer move.](../virtualreality/oculus.md).
\ No newline at end of file
+You have actually followed the dog embodiment of Medusa,
+and you made eye contact. [You can no longer move.](../virtualreality/oculus.md).
+
+You notice she is chewing on a pink sock. You decide to [check out the sock](fill-sock.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.